huridocs/uwazi
# removed redux-form as dependency

Fixes issue #23

PR check list:


QA check list:

- [ ] Smoke test the functionality described in the issue
- [ ] UI responsiveness
- [ ] Tested in a browser other than chrome

NOTE: Make sure the build passes.
